### PR TITLE
[SYCL][InvokeSimd][E2E] Enable uniform retval tests

### PR DESCRIPTION
**Pull request by sarnex**
They pass with the new driver.

### CODE REVIEW

File: sycl/test-e2e/InvokeSimd/Spec/ImplicitSubgroup/uniform_retval.cpp
Comment (aelovikov-intel) [inline sycl/test-e2e/InvokeSimd/Spec/ImplicitSubgroup/uniform_retval.cpp:2]: I'm afraid it will enable Win as well, and there was no update there. Need to wait for the CI to finish before merging.

Comment (sarnex) [reply sycl/test-e2e/InvokeSimd/Spec/ImplicitSubgroup/uniform_retval.cpp:2]: We don't run these on Windows, see [here](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/InvokeSimd/lit.local.cfg)

**Review by v-klochkov — APPROVED** [review]

**Comment by sarnex** [comment]
Fails not related, it's from me destroying the repo in another commit :)

**Pull request merged**
